.github/workflows/repo_file_for_test_cpp_package.repos

```diff
@@ -1,10 +1,10 @@
 # repo file only containing osrf_testing_tools_cpp
 # This repository has been chosen because it does depend on anything, as of
-# 1.2.1.
+# 1.3.4.
 #
 # https://github.com/ament/ament_lint/tree/0.8.1
 repositories:
   osrf_testing_tools_cpp:
     type: git
     url: https://github.com/osrf/osrf_testing_tools_cpp.git
-    version: 1.2.1
+    version: 1.3.4
```

Comment:
uprev from 1.2.1 to 1.3.4 to address https://github.com/osrf/osrf_testing_tools_cpp/issues/35 on windows and some macos runners. 

@jacobperron is there a reason that you tagged the latest release at https://github.com/osrf/osrf_testing_tools_cpp/tags to be 1.3.4, when the release before that was 1.4?

Reply:
1.3.4 is for Foxy (the tag is only on the [foxy branch](https://github.com/osrf/osrf_testing_tools_cpp/commits/foxy)).
1.4.0 is the latest tag for Galactic and Rolling.